streamly.cabal

```diff
@@ -364,14 +366,15 @@ library
                     -- constraints of these libraries should match with
                     -- the GHC versions we support
                        base              >= 4.8   &&  < 5
+                     , ghc
```

Comment:
```suggestion
```

Reply:
Will delete the commit containing this change in the end. There is still a little to be done.